string color
int16 min_size
int16 max_size
---
geometry_msgs/Pose2D pose
int16 size
